Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -677,10 +675,4 @@ def adjoint(self, x, out=None):
     def is_linear(self):
         return self.linear_flag
 
-    def calculate_norm(self):
-        '''Returns the norm of the CompositionOperator, that is the product of the norms
-        of its operators.'''
-        norm = 1.
-        for operator in self.operators:
-            norm *= operator.norm()
-        return norm

```

Comment:
Is this wrong?

If not, I don't understand why would we want to remove this, because doing the `PowerMethod` on the whole `CompositionOperator` is not going to cache the norms of the operators.

Reply:
> Is this wrong?

In general, yes, it's only an upper bound, as stated in #1632. (take an example where one operator multiplies all dimensions except the first with 0, and the other multiplies the first with 0. The product of the 2 multiplies everything with 0, and hence has norm 0).

There's no general way to discover if the upper bound is correct or not. If you want `norm()` to return the actual norm, this is necessary (unfortunately).